blog/2025/rippled-2.6.0.md

```diff
@@ -0,0 +1,153 @@
+    editPage:
+        hide: true
+---
+# Introducing XRP Ledger version 2.6.0
+
+Version 2.6.0 of `rippled`, the reference server implementation of the XRP Ledger protocol, is now available. This release adds new features and bug fixes.
+
+
+## Action Required
+
+If you run an XRP Ledger server, upgrade to version 2.6.0 as soon as possible to ensure service continuity.
+
+
+## Install / Upgrade
+
+On supported platforms, see the [instructions on installing or updating `rippled`](../../docs/infrastructure/installation/index.md).
+
+| Package | SHA-256 |
+|:--------|:--------|
+| [RPM for Red Hat / CentOS (x86-64)](https://repos.ripple.com/repos/rippled-rpm/stable/rippled-2.6.0-1.el9.x86_64.rpm) | `efd0d61ce0906af1862eeb0691717ddc546335294dc2ce8db1d9e243d215846c` |
+| [DEB for Ubuntu / Debian (x86-64)](https://repos.ripple.com/repos/rippled-deb/pool/stable/rippled_2.6.0-1_amd64.deb) | `82570dd0c36d3ce2f79902ae99c44bcfe710d619d806805a5af9d8f56b7dbea1` |
+
+For other platforms, please [build from source](https://github.com/XRPLF/rippled/blob/master/BUILD.md). The most recent commit in the git log should be the change setting the version:
+
+```text
+commit 2df7dcfdebcb0cdbd030c1f4b09ac748af95659c
+Author: Michael Legleux <mlegleux@ripple.com>
+Date:   Wed Aug 27 10:25:53 2025 -0700
+
+    Set version to 2.6.0
+```
+
+
+## Full Changelog
+
+### Amendments
+
+- None
+
+
+### New Features
+
+- Improved the `account_tx` method to show MPT-related transactions for an issuer. ([#5530](https://github.com/XRPLF/rippled/pull/5530))
+- Added the `network_id` field to `validations` and `ledger` subscription streams. ([#5579](https://github.com/XRPLF/rippled/pull/5579))
+- Optimized hash performance. ([#5469](https://github.com/XRPLF/rippled/pull/5469))
+
+
+### Bug Fixes
+
+- Added `nftoken_id`, `nftoken_ids`, and `offer_id` to meta field of transaction streams. ([#5230](https://github.com/XRPLF/rippled/pull/5230))
+- Fixed a crash when trace-logging in tests. ([#5529](https://github.com/XRPLF/rippled/pull/5529))
+- Removed circular includes in header files. ([#5544](https://github.com/XRPLF/rippled/pull/5544))
+- Fixed Boost Library linking to be explicit. ([#5546](https://github.com/XRPLF/rippled/pull/5546))
+- Added the `allowTrustLineLocking` flag as an option for the `account_info` method. ([#5535](https://github.com/XRPLF/rippled/pull/5525))
+- Silenced a dummy warning in GCC 12. ([#5554](https://github.com/XRPLF/rippled/pull/5554))
+- Removed the automatic creation of trust lines to the `Destination` account in `VaultWithdraw` transactions. ([#5572](https://github.com/XRPLF/rippled/pull/5572))
+- Fixed an issue where the order of `PriceDataSeries` desynced between the time a `PriceOracle` was created and when it was updated. ([#5485](https://github.com/XRPLF/rippled/pull/5485))
+- Fixed stalled consensus detection to prevent false positives in situations where there were no disputed transactions. ([#5627](https://github.com/XRPLF/rippled/pull/5627))
```

Comment:
I'm not sure what order these notes are in, if any, but this one seems important enough to put higher in the list.

Reply:
Reordered the list; moved ledger fixes up and build errors/logging errors down.